supports/support-proc-macro/src/lib.rs

```diff
@@ -26,14 +28,15 @@ mod crypto;
 /// ```
 /// This will be expand to
 /// ```rust
+/// use support_proc_macro::BridgeCrypto;
+///
 /// pub struct Foo {
-///     #[crypto(is_enable)]
+///     // #[crypto(is_enable)]
```

Comment:
Why?

Reply:
This attribute should indeed be in the output, but happen an error, 

```rust
failures:

---- src\lib.rs - derive_bridge_crypto (line 30) stdout ----
error: cannot find attribute `crypto` in this scope
 --> src\lib.rs:34:7
  |
7 |     #[crypto(is_enable)]
  |       ^^^^^^

error: aborting due to previous error
```

I tried `use support_proc_macro::*` it's also error, so comment this first.

Reply:
how about

```
#[macro_use]
extern crate support_proc_macro;
```

Reply:
```
running 4 tests
test src\lib.rs - derive_bridge_crypto (line 74) ... FAILED
test src\lib.rs - derive_bridge_crypto (line 61) ... ok
test src\lib.rs - derive_bridge_crypto (line 30) ... ok
test src\lib.rs - derive_bridge_crypto (line 14) ... ok

failures:

---- src\lib.rs - derive_bridge_crypto (line 74) stdout ----
error[E0468]: an `extern crate` loading macros must be at the crate root
 --> src\lib.rs:76:1
  |
4 | extern crate support_proc_macro;
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: cannot find attribute `crypto` in this scope
 --> src\lib.rs:79:7
  |
7 |     #[crypto(decrypt)]
  |       ^^^^^^

error: cannot find attribute `crypto` in this scope
 --> src\lib.rs:81:7
  |
9 |     #[crypto(decrypt)]
  |       ^^^^^^

error: aborting due to 3 previous errors

For more information about this error, try `rustc --explain E0468`.
Couldn't compile the test.

failures:
    src\lib.rs - derive_bridge_crypto (line 74)

test result: FAILED. 3 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.70s

error: test failed, to rerun pass '--doc'
```

Reply:
Actually, these are expanded codes. So remove them.